packages/create-react-app/index.js

```diff
@@ -111,20 +147,22 @@ function createApp(name, verbose, version, template) {
   checkAppName(appName);
   fs.ensureDirSync(name);
   if (!isSafeToCreateProjectIn(root)) {
-    console.log('The directory ' + chalk.green(name) + ' contains files that could conflict.');
+    console.log(
+      'The directory ' +
+        chalk.green(name) +
+        ' contains files that could conflict.'
```

Comment:
Related to https://github.com/jlongster/prettier/issues/489 and https://github.com/jlongster/prettier/pull/670